examples/advanced-web-service/__snapshots__/chart.test.ts.snap

```diff
@@ -185,7 +187,7 @@ proxy_cookie_path / "/$cookie_path_patches";
         "region": "local",
         "service": "advanced-development-local",
       },
-      "name": "nginx-config-82htd5f6t7",
+      "name": "nginx-config-t6227t7d4g",
```

Comment:
question: I am not sure what I have done to trigger a change here. Is this expected?